master/internal/configpolicy/utils.go

```diff
@@ -94,7 +94,7 @@ func ValidateExperimentConfig(
 		if cp.InvariantConfig.RawResources != nil {
 			checkAgainstGlobalPriority(priorityEnabledErr, cp.InvariantConfig.RawResources.RawPriority)
 			if err := checkConstraintConflicts(cp.Constraints, cp.InvariantConfig.RawResources.RawMaxSlots,
-				cp.InvariantConfig.RawResources.RawSlotsPerTrial, cp.InvariantConfig.RawResources.RawPriority); err != nil {
+				cp.InvariantConfig.RawResources.RawSlots, cp.InvariantConfig.RawResources.RawPriority); err != nil {
```

Comment:
I don't understand this change. Experiments use slots per trial, not slots, right?

Reply:
If that's the case, @amandavialva01 can you also update the logic [here](https://github.com/determined-ai/determined/blob/9efd96df058d33c4e7524e486c0629aea1283aa5/master/internal/configpolicy/task_config_policy.go#L91) to use slots per trial?

Reply:
_Ooh_ my mistake! Changed this. I think the actual discrepancy was coming from the < / > issue that Kristine mentioned

Reply:
Synced with Bradley - for NTSC we'll compare "slots" to "constraints.resources.max_slots" and for experiments we'll compare "slotsPerTrial" to "constraints.resources.max_slots".

Reply:
Makes sense! Yea I just saw the comment in the `ResourcesConfig` struct, somehow glossed over that before lol